src/firefly/java/edu/caltech/ipac/visualize/resources/Footprint_JWST.reg

```diff
@@ -0,0 +1,67 @@
+#FGS
+POLYGON  0.03794999  -0.17407720   0.03843333  -0.21384895   0.07796106  -0.21439047   0.07665551  -0.17417432   0.03794999  -0.17407720 # color=cyan tag={FGS}
+POLYGON -0.01246111  -0.17438280  -0.01299444  -0.21437121   0.02619722  -0.21371843   0.02595278  -0.17398279  -0.01246111  -0.17438280 # color=cyan tag={FGS}
+#MIRI
+POLYGON   -0.13494975  -0.08756358  -0.13742196  -0.11857171  -0.10607210  -0.12121628  -0.10358878  -0.08992200  -0.13494975  -0.08756358 # color=yellow tag={MIRI}
+POLYGON   -0.13951917  -0.08917466  -0.14044138  -0.08903022  -0.14029416  -0.08801078  -0.13937751  -0.08814134  -0.13951917  -0.08917466 # color=yellow tag={MIRI}
+POLYGON   -0.13951917  -0.08921356  -0.14045528  -0.08907188  -0.14030250  -0.08805522  -0.13937473  -0.08818856  -0.13951917  -0.08921356 # color=yellow tag={MIRI}
+POLYGON   -0.13951639  -0.08913022  -0.14041917  -0.08899411  -0.14026638  -0.08797745  -0.13936639  -0.08810245  -0.13951639  -0.08913022 # color=yellow tag={MIRI}
+POLYGON   -0.13943306  -0.08944411  -0.14059694  -0.08928577  -0.14038027  -0.08797744  -0.13923306  -0.08814412  -0.13943306  -0.08944411 # color=yellow tag={MIRI}
+POLYGON   -0.13947472  -0.08951077  -0.14063860  -0.08935521  -0.14041917  -0.08804411  -0.13927195  -0.08821356  -0.13947472  -0.08951077 # color=yellow tag={MIRI}
+POLYGON   -0.13941917  -0.08949410  -0.14056639  -0.08934133  -0.14035249  -0.08803022  -0.13920806  -0.08819967  -0.13941917  -0.08949410 # color=yellow tag={MIRI}
+POLYGON   -0.13937751  -0.08964411  -0.14094694  -0.08941355  -0.14077194  -0.08770800  -0.13922195  -0.08790245  -0.13937751  -0.08964411 # color=yellow tag={MIRI}
+POLYGON   -0.13938306  -0.08969967  -0.14096360  -0.08946632  -0.14076916  -0.08776356  -0.13922195  -0.08795800  -0.13938306  -0.08969967 # color=yellow tag={MIRI}
+POLYGON   -0.13938028  -0.08969967  -0.14092749  -0.08946632  -0.14073861  -0.08776356  -0.13921362  -0.08795800  -0.13938028  -0.08969967 # color=yellow tag={MIRI}
+POLYGON   -0.13901084  -0.09011910  -0.14103860  -0.08987188  -0.14059694  -0.08768578  -0.13863306  -0.08800800  -0.13901084  -0.09011910 # color=yellow tag={MIRI}
+POLYGON   -0.13901917  -0.09009966  -0.14104138  -0.08985798  -0.14057749  -0.08767189  -0.13861639  -0.08798856  -0.13901917  -0.09009966 # color=yellow tag={MIRI}
+POLYGON   -0.13902472  -0.09008299  -0.14099972  -0.08982187  -0.14058027  -0.08765244  -0.13862751  -0.08797467  -0.13902472  -0.09008299 # color=yellow tag={MIRI}
+POLYGON  360.00148056  -0.00349167 359.98601112  -0.01929166 359.97019723  -0.00381111 359.98566389   0.01199167 360.00148056  -0.00349167 # color=yellow tag={MIRI}
```

Comment:
those 3 polygon belongs to WFPC2. remove them and check that MIRI and WFPC2 is correctly defined.